src/app/content/practiceQuestions/components/EmptyScreen.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { useDispatch } from 'react-redux';
+import styled from 'styled-components/macro';
+import { textRegularStyle } from '../../../components/Typography';
+import { LinkedArchiveTreeSection } from '../../types';
+import { setSelectedSection } from '../actions';
+import NextSectionMessage from './NextSectionMessage';
+
+// tslint:disable-next-line: variable-name
+const StyledEmptyScreen = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  max-width: 38rem;
+  margin: 2rem auto;
+  flex: 1;
+  text-align: center;
+  ${textRegularStyle}
+
+  > span {
```

Comment:
Is this for the plain span in line `35`? I think it's always better to make it a styled component and if it's very specific, use the respective styled component's name inside the parent styles.